pyo3-bytes/src/bytes.rs

```diff
@@ -93,17 +95,24 @@ impl PyBytes {
     // By setting the argument to PyBytes, this means that any buffer-protocol object is supported
     // here, since it will use the FromPyObject impl.
     #[new]
+    #[pyo3(signature = (buf = PyBytes(Bytes::new())))]
```

Comment:
This is the signature shown to Python, right? This isn't valid Python code.

Reply:
No I don't think it is. IIRC `__text_signature__` is?